code/datums/mutantraces.dm

```diff
@@ -1156,6 +1208,11 @@
 	icon = 'icons/mob/monkey.dmi'
 	icon_state = "seamonkey"
 	aquatic = 1
+	tail = SEAMONKEY_TAIL
+
+	New()
+		..()
```

Comment:
probably be better to do `. = ..()` for these

Reply:
I don't know what that does, but I'll do that once I do.

Reply:
I ended up taking out those New() procs. Turns out, I didn't need to call set_tail in there!